TestIssuesFromArgsWithFields/multiple_issues is flaky probably due to parallelism

I'll fill this in in a moment, just doing a release:

https://github.com/cli/cli/actions/runs/5123166785/jobs/9213288152

```
--- FAIL: TestIssuesFromArgsWithFields (0.00s)
    --- FAIL: TestIssuesFromArgsWithFields/multiple_issues (0.00s)
        lookup_test.go:290: want issue #1, got #2
        lookup_test.go:290: want issue #2, got #1
```

--- a/pkg/cmd/issue/shared/lookup_test.go
+++ b/pkg/cmd/issue/shared/lookup_test.go
@@ -285,10 +285,8 @@ func TestIssuesFromArgsWithFields(t *testing.T) {
 				return
 			}
 			assert.NoError(t, err)
-			for i, issue := range issues {
-				if issue.Number != tt.wantIssues[i] {
-					t.Errorf("want issue #%d, got #%d", tt.wantIssues[i], issue.Number)
-				}
+			for i := range issues {
+				assert.Contains(t, tt.wantIssues, issues[i].Number)
 			}
 			if repo != nil {
 				repoURL := ghrepo.GenerateRepoURL(repo, "")

